src/gitignore.ts

```diff
@@ -0,0 +1,36 @@
+/**
+ * @license
+ * Copyright (c) 2018 The Polymer Project Authors. All rights reserved.
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt
+ * The complete set of authors may be found at
+ * http://polymer.github.io/AUTHORS.txt
+ * The complete set of contributors may be found at
+ * http://polymer.github.io/CONTRIBUTORS.txt
+ * Code distributed by Google as part of the polymer project is also
+ * subject to an additional IP rights grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+
+import * as fse from 'fs-extra';
+import {EOL} from 'os';
+
+const nodeModulesLine = 'node_modules';
+const searchNodeModulesLine = new RegExp(`^${nodeModulesLine}`);
```

Comment:
I've seen a number of variations used, like `/node_modules` to limit to top level. Maybe do `^$\/?{nodeModulesLine}`?

Reply:
sure, will do